app/src/main/java/com/weatherxm/ui/devicealerts/DeviceAlertsActivity.kt

```diff
@@ -74,10 +74,10 @@ class DeviceAlertsActivity : BaseActivity(), DeviceAlertListener {
     }
 
     override fun onLowBatteryReadMoreClicked() {
-        val url = if (device?.profile == DeviceProfile.M5) {
-            getString(R.string.docs_url_low_battery_m5)
-        } else {
-            getString(R.string.docs_url_low_battery_helium)
+        val url = when (device?.profile) {
+            DeviceProfile.M5 -> getString(R.string.docs_url_low_battery_m5)
+            DeviceProfile.D1 -> getString(R.string.docs_url_low_battery_d1)
+            else -> getString(R.string.docs_url_low_battery_helium)
```

Comment:
I would prefer to check explicitly for helium and return an empty string in unhandled cases.

Reply:
Valid point, I'll fix it.